.changeset/moody-tips-give.md

```diff
@@ -0,0 +1,7 @@
+---
+'@finos/legend-extension-dsl-persistence': patch
+'@finos/legend-application-studio': patch
+'@finos/legend-graph': patch
+---
+
+Handle testable refactor
```

Comment:
I think we can give a better description here. Let's give maybe one or two sentences on the main model changes we are doing on Testables. Thanks!